src/Microsoft.Framework.PackageManager/List/DependencyListOperation.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using Microsoft.Framework.Runtime;
+
+namespace Microsoft.Framework.PackageManager.List
+{
+    internal class DependencyListOperation
+    {
+        private const string LibraryTypeProject = "Project";
+        private const string Configuration = "Debug";
+
+        private readonly FrameworkName _framework;
+        private readonly DependencyListOptions _options;
+        private readonly GacDependencyResolver _gacResolver;
+        private readonly ApplicationHostContext _hostContext;
+
+        public DependencyListOperation(DependencyListOptions options, FrameworkName framework)
+        {
+            _options = options;
+            _framework = framework;
+
+            _hostContext = CreateApplicationHostContext();
+            _gacResolver = new GacDependencyResolver();
+        }
+
+        public bool Execute()
+        {
+            var nonProjectLibs = _hostContext.LibraryManager.GetLibraries()
```

Comment:
That's what you suggested. We left out the projects since they aren't resolved to any local assemblies.